Kickstarter-iOS/Views/Cells/SettingsTextInputCell.swift

```diff
@@ -43,6 +52,9 @@ final class SettingsTextInputCell: UITableViewCell {
   override func bindStyles() {
     super.bindStyles()
 
+    _ = self
```

Comment:
This PR adds the ability to tap the cell anywhere which makes the text field first responder (the side effect was that it also highlighted the cell)...this is a standard way of disabling selection mode on the cell.